modules/shared/src/test/scala/org/tessellation/security/signature/SignedValidatorSuite.scala

```diff
@@ -92,6 +100,77 @@ object SignedValidatorSuite extends MutableIOSuite {
     } yield expect.same(SignedValidator.SignersNotInSeedlist(NonEmptySet.one(peerId2.toId)).invalidNec, result)
   }
 
+  test("validateSignedByMajority should succeed when there is no seedlist") { res =>
+    implicit val (h, sp) = res
+
+    forall(signedOf(addressGen)) { signedAddress =>
+      IO {
+        val actual = mkValidator().validateSignedByMajority(None, signedAddress)
+        val expected = signedAddress.validNec
+        expect.same(actual, expected)
+      }
+    }
+  }
+
+  test("validateSignedByMajority should fail on empty seedlist") { res =>
```

Comment:
I think that the empty seedlist is the same case as the next test, but there is no test for no seedlist.

Reply:
@marcinwadon I don't understand your concern. There is this test, "validateSignedByMajority should succeed when there is no seedlist" -- are you saying it should fail in that case?

Reply:
> There is this test, "validateSignedByMajority should succeed when there is no seedlist"
sorry, I missed that one. it's fine then :+1: